checkbox-ng/checkbox_ng/config.py

```diff
@@ -68,6 +67,9 @@ def load_configs(launcher_file=None):
             config = expand_all(os.path.join(d, config_filename))
             if os.path.exists(config):
                 configs.append(config)
+    # Add config from launcher last so it gets precedence over others
+    if launcher_file:
+        configs.append(launcher_file)
```

Comment:
For the record, the ConfigParser read() works this way:

It is possible to read several configurations into a single [ConfigParser](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser), where the most recently added configuration has the highest priority. Any conflicting keys are taken from the more recent configuration while the previously existing keys are retained.